config/config.go

```diff
@@ -5,6 +5,14 @@ import (
 	"github.com/madflojo/hord/databases/cassandra"
 )
 
+var (
+	// For log use:
+	GrpcKeyNotDefine       = "Key is not defined within request"
```

Comment:
Rather than putting these in config maybe these should be error types within the grpc file itself? What do you think?

Reply:
you might be right , it is probably easier to manage them if they are in related files 

i make those changes